java/google/registry/module/backend/BackendComponent.java

```diff
@@ -69,6 +70,7 @@
         UrlFetchTransportModule.class,
         UseAppIdentityCredentialForGoogleApisModule.class,
         VoidDnsWriterModule.class,
+        FlowInjectionModule.class,
```

Comment:
I'm curious why it was necessary to add this to the service-level components.  Why not FlowComponent?  If it's in FlowComponent, is it actually needed in anything else?